requirements.dev.txt

```diff
@@ -4,6 +4,7 @@ bandit==1.7.0
 black==20.8b1
 codecov==2.1.11
 darglint==1.7.0
```

Comment:
Q: What is the difference between ```darglint``` and ```docformatter```?

Reply:
Is ```docformatter``` something that it is used with ```darglint```?

Reply:
@gmuraru, darglint/pydocstyle checks the documentation, docformatter formats your code.
Darglint ~ flake, docformatter ~ black.

This kind of things helps to keep the code well-formed, but at the beginning can be a bit annoying if the developer is not familiar with pep conventions. I will discuss with the `docs_team` if this should be (or not) an standard for every openmined project.